### PR TITLE
To Do List Workspace

### PR DESCRIPTION
**Pull request by AmaliaGM**
## Project Requirements

- Set up a new project with webpack that is based on the webpack exercise you have already completed.
- Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element). - The index.html file must be set as a template using the HTML Webpack Plugin.
- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.

- All the source files (index.html, index.js and style.css) must be located in /src directory and the distribution files will be generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    this.chore = chore;
+    this.complete = complete;
+  }
+}
+const add = document.querySelector('#add');
+add.addEventListener('click', () => {
+  const ID = itemID();
+  const chore = document.querySelector('#todoInput').value;
+  const complete = document.querySelector('.completed');
+  const newItem = new AddItem(ID, chore, complete);
```
Comment (Mihndim2020) [inline src/index.js:20]: - [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md) for this project, " `Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].` ", please ensure the value of the `completed key` is boolean. Something like so: 
```
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
```
Currently the value is `null`, please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @AmaliaGM, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Your project is running free of `Linter errors`
:heavy_check_mark: Great job with Webpack

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION


# Status: Requires Changes ♻️
_Check the comments under the review._

 #### I will leave the comments here, since your `Readme file` is not accessible for this `PR` as you did not make any change on it ####

- [ ] Please kindly update your `readme file` to reflect the changes for this milestone as this will make your `readme file` more informative and professional. Checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project. Replace all the information in the various sections of the template with information pertaining to your project. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShoiraTa** [comment]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @AmaliaGM , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (AmaliaGM) [reply src/index.js:20]: Thanks for your review! Consider changes made

**Comment by AmaliaGM** [comment]
@ShoiraTa Thanks for your feedback! Have a really nice day!

**Pull request merged**